include/gridtools/common/demangle.hpp

```diff
@@ -0,0 +1,118 @@
+// core::demangle
```

Comment:
I can likely format this whole file, yes? Diffs with the original become more difficult, but considering that the original hasn't changed in years I think it's fair to say this is unlikely to ever be updated. And since it's a small file manually diffing isn't too bad...

Reply:
Agree